### DISCUSSION TITLE
"Ideal" predicted phosphoDIA library

### DISCUSSION THREAD

**Discussion by seanso95** [vdemichev/DiaNN#1096] [Q&A] [answered]
Hi all,

I am new to proteomics and excited to try out DIA-NN v1.9 with phosphoproteomic DIA data!

I have been reading the guidelines to DIA-NN in the GitHub and am unsure of the best way to initialise a predicted library of phosphoproteomic data. My sample contains variable PTMs of Ox(M), Ac(N-term), and Phospho(STY) - though I am most specifically interested in phospho modifications. In the "Getting started" section, it is suggested to load the species fasta file and generate a library with default settings, selecting PTMs where necessary (I selected all the variable PTMs I mentioned above). In a test ultra-fast run, I was able to detect some phospho signatures, which was super cool!

However, the section on PTMs suggests to alter the default settings by selecting _only_ phospho, changing the maximum number of variable modifications to 3, and setting precursor charge range from 2-3. Given that my sample contains Ox(M) and Ac(N-term) PTMs as well, should I additionally add these 2 PTMs and continue with the suggested settings? Not sure what is a sensible predicted library here.

Thank you for any help you may provide!
Sean

**Comment by vdemichev** [comment, marked as answer]
Hi Sean,

Please try first as suggested, and then see if you can improve the results by adding other modifications (In particular, Ac (protein N-term) is usually not harmful, as it does not expand the search space a lot).

> Given that my sample contains Ox(M) and Ac(N-term) PTMs as well

However, Ox(M) is a common technical artifact. As such, it's might be a bad idea to use it for quantification. In terms of peptide identification, usually can identify also without Ox(M). So including Ox(M) typically makes little sense, unless you have really a lot of oxidations.

Best,
Vadim

**Reply by seanso95** [reply]
Thank you Vadim for the prompt reply! I will do these changes as suggested.

By the way, hoping you can advise: would it be expected that the addition of the N-term Ac or 2 missed cleavages would be more impactful for identification (for prioritising which spectral library I test next)?

**Reply by vdemichev** [reply]
Can add both at the same time (so long as you add Ac only for protein N-term - please use the GUI checkbox for that)

**Reply by seanso95** [reply]
Awesome - thank you!

Before I close and mark the topic as completed (and thank you for all the amazing help and beautiful software!!!), could you please advise on how DIA-NN handles multiple libraries?

For example, I have:
1. offline fractionated DDA libraries (analysed by MaxQuant)
2. results from DDA experiments (analysed by MaxQuant; can use the spectral outputs as input libraries)
3. in silico predicted library from DIA-NN v1.9.1

If I were to specify all three libraries in the GUI, how would DIA-NN handle multiple entries for the same phosphopeptides? Is there a best-practice way to merge these?

Thank you,
Sean

**Reply by vdemichev** [reply]
Hi Sean,

You can load multiple libraries (either all .tsv or all .parquet) using multiple --lib commands, here DIA-NN choses either the first it encounters or the most confident (if q-values are present in the library) entry per precursor. But this is only OK if they are 'homogeneous', i.e. eveyrthying about them should be the same. For example, combining libs from different offline frac experiments is OK. 

Best,
Vadim

**Reply by seanso95** [reply]
Hi Vadim,

Thank you for the response. I'm glad to report that using the predicted phospho speclib with the standard phospho settings gave us quite good phosphopeptide detection, and that adjusting the precursor charge range and adding N-acetylation boosted this by ~3fold (interestingly).

I will mark the question as answered, though I will note that I think there is some sort of bug with msms.txt MaxQuant speclib conversion (which I saw you have noted will be fixed in 1.9.2). As such, I have not tried to merge the msms.txt with the predicted speclib generated by a FASTA digest.

Thank you!
Sean

**Reply by sooheon** [reply]
@sartoriius did you adjust precursor charge range to be wider than 2-3?

**Reply by seanso95** [reply]
Hi @sooheon,

Yes, I adjusted my charge range from 2-4. This was because we have paired DDA samples for all our DIA experiments (some designed as offline fractionated DDA-based spectral libraries, some as actual DDA-capture experiments directly matching DIA-capture experiments). In my case, ~10% of the DDA reads were from precursors with 4 charges; hence the expansion of the charge range.
